infra/data-pipeline.yml

```diff
@@ -442,8 +442,8 @@ tasks:
       command:
         - "/bin/sh"
         - "-lcxe"
-        - "taskboot deploy-heroku --artifact-filter public/bugbug/bugbug-http-service.tar --heroku-app bugbug &&
-           taskboot deploy-heroku --artifact-filter public/bugbug/bugbug-http-service-bg-worker.tar --heroku-app bugbug --heroku-dyno-type worker"
+        - "taskboot deploy-heroku --artifact-filter public/bugbug/registry.hub.docker.com_mozilla_bugbug-http-service_${version}.tar --heroku-app bugbug &&
+           taskboot deploy-heroku --artifact-filter public/bugbug/registry.hub.docker.com_mozilla_bugbug-http-service-bg-worker_${version}.tar --heroku-app bugbug --heroku-dyno-type worker"
```

Comment:
What if we did *-http-service_* and *-http-service-bg-worker_*?

Reply:
I preferred to be very precise in the artifacts we would download as we could get any artifacts from a dependency that match the name